### ISSUE TITLE
"console movetoavatarcache" error

### ISSUE THREAD

**Issue by SpcCw** [friendica/friendica#11560]
- [ o ] I have searched open and closed issues for duplicates

I decided to try [new console command](https://github.com/friendica/friendica/pull/11549) moving avatars to cache and got an error:

```
bin/console movetoavatarcache
1/146478	39	https://kocinski.me/friendica	1234PHP Notice:  Undefined index: thumb in /usr/share/nginx/friends/src/Contact/Avatar.php on line 242
[Error] Argument 1 passed to Friendica\Contact\Avatar::deleteCacheFile() must be of the type string, null given, called in /usr/share/nginx/friends/src/Contact/Avatar.php on line 242
```

Now this is probably because I use filesystem storage and don't have anything in DB?

Friendica: current RC
PHP: 7.4.29

**Comment by annando** [comment]
No, it's just because this is a brand new feature and obviously needs some refinement.

**Comment by SpcCw** [comment]
I noticed it a bit too late so just FYI: after I ran the command after this update some of local users lost their avatars - now there's is default "person_300" avatar in their profiles and resized versions are non-existent, avatar images are missing from photo albums as well.

Reuploading or setting avatar from existing pictures works though.

What is strange it happened only to users who did not log in for a while. E.g. I have about 20 RSS users which were created a year ago and since then only "posted" from mirrored RSS contacts - all of them lost avatars.
